command/auth_enable.go

```diff
@@ -160,6 +162,13 @@ func (c *AuthEnableCommand) Flags() *FlagSets {
 		Usage:   "Enable seal wrapping of critical values in the secrets engine.",
 	})
 
+	f.IntVar(&IntVar{
+		Name:    "version",
+		Target:  &c.flagVersion,
+		Default: 0,
```

Comment:
So the version will be `0` for "old" kvs, and `2` for the new one?

Reply:
I think those will be interchangeable, with zero being interpreted as "not set"

Reply:
Yep! What jeff said